netfilterqueue/_impl.pyx

```diff
@@ -4,9 +4,16 @@ function.
 
 Copyright: (c) 2011, Kerkhoff Technologies Inc.
 License: MIT; see LICENSE.txt
+
+Expanded features and performance improvements from downstream development
+of DNXFIREWALL.
+ - DOWRIGHT @ Wright Network Solutions, LLC.
 """
 
 # Constants for module users
+import Cython
```

Comment:
These imports are placed awkwardly relative to the comment that is now above them but describes the constants below